src/script/externalRoute.ts

```diff
@@ -20,79 +20,56 @@
 import {currentLanguage} from './auth/localeConfig';
 import {Config} from './Config';
 
-const env = window.wire.env;
-
-export const URL = {
-  ACCOUNT: env.URL?.ACCOUNT_BASE,
-  PRIVACY_POLICY: env.URL?.PRIVACY_POLICY,
-  SUPPORT: env.URL?.SUPPORT.INDEX,
-  TEAM_SETTINGS: env.URL?.TEAMS_BASE,
-  TERMS_OF_USE_PERSONAL: env.URL?.TERMS_OF_USE_PERSONAL,
-  TERMS_OF_USE_TEAMS: env.URL?.TERMS_OF_USE_TEAMS,
-  WEBAPP: {
-    INTERNAL: 'https://wire-webapp-staging.wire.com',
-    PRODUCTION: env.APP_BASE || 'https://app.wire.com',
-    STAGING: 'https://wire-webapp-staging.zinfra.io',
-  },
-  WEBSITE: env.URL?.WEBSITE_BASE,
-};
-
-export const URL_PATH = {
-  CREATE_TEAM: '/create-team/',
-  DECRYPT_ERROR_1: '/articles/207948115',
-  DECRYPT_ERROR_2: '/privacy/error-2/',
-  MANAGE_SERVICES: '/services/',
-  MANAGE_TEAM: '/login/',
-  PASSWORD_RESET: '/forgot/',
-  PRIVACY_HOW: '/privacy/how/',
-  PRIVACY_UNVERIFIED_USERS: '/articles/202857164',
-  PRIVACY_WHY: '/articles/207859815',
-  SUPPORT_USERNAME: '/support/username/',
-} as const;
+const URL = Config.getConfig().URL;
 
 const getTeamSettingsUrl = (path: string = '', utmSource?: string): string | undefined => {
   const query = utmSource ? `?utm_source=${utmSource}&utm_term=desktop` : '';
-  const teamSettingsUrl = `${URL.TEAM_SETTINGS}${path}${query}`;
-  return URL.TEAM_SETTINGS ? teamSettingsUrl : undefined;
+  const teamSettingsUrl = `${URL.TEAMS_BASE}${path}${query}`;
+  return URL.TEAMS_BASE ? teamSettingsUrl : undefined;
 };
 
-export const getWebsiteUrl = (path: string = '', pkCampaign?: string): string | undefined => {
-  if (URL.WEBSITE) {
+const getWebsiteUrl = (path: string = '', pkCampaign?: string): string | undefined => {
+  if (URL.WEBSITE_BASE) {
     const query = pkCampaign ? `?pk_campaign=${pkCampaign}&pk_kwd=desktop` : '';
-    const websiteUrl = `${URL.WEBSITE}${path}${query}`;
-    return addLocaleToUrl(URL.WEBSITE ? websiteUrl : undefined);
+    const websiteUrl = `${URL.WEBSITE_BASE}${path}${query}`;
+    return addLocaleToUrl(URL.WEBSITE_BASE ? websiteUrl : undefined);
   }
   return undefined;
 };
 
-const getHelpCenterUrl = (path: (typeof URL_PATH)[keyof typeof URL_PATH]) => {
-  if (URL.SUPPORT) {
-    const helpcenterUrl = `${URL.SUPPORT}${path}`;
-    return addLocaleToHelpCenterUrl(URL.SUPPORT ? helpcenterUrl : undefined);
-  }
-  return undefined;
+const getAccountPagesUrl = (path: string = ''): string | undefined => {
+  const accountPagesUrl = `${URL.ACCOUNT_BASE}${path}`;
+  return URL.ACCOUNT_BASE ? accountPagesUrl : undefined;
```

Comment:
You can simplify it :) 

```suggestion
  return URL.ACCOUNT_BASE ? `${URL.ACCOUNT_BASE}${path}` : undefined;
```